scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -420,7 +420,7 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
 
             $rootScope.publishAfterTansa = function () {
                 if (!onlyTansaProof) {
-                    $scope.saveTopbar().then(continueAfterPublish ? $scope.publishAndContinue : $scope.publish);
+                    $scope.saveTopbar().then(continueAfterPublish === true ? $scope.publishAndContinue : $scope.publish);
```

Comment:
I'm just curious: why does this fix the problem? What other truthy values can `continueAfterPublish` have that cause issues here?